src/Sentry/Laravel/Tracing/CallableProxy.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Laravel\Tracing;
+
+use Illuminate\Contracts\Container\Container;
+use ReflectionClass;
+use ReflectionMethod;
+use Sentry\Tracing\SamplingContext;
+
+class CallableProxy
+{
+    /**
+     * @var Container
+     */
+    private $container;
+
+    public function __construct(Container $container)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * @param array|callable|null $tracesSampler
+     *
+     * @return array|callable
```

Comment:
```suggestion
     * @param array|callable|null $tracesSampler
     *
     * @return array|callable|null
```

Reply:
Im gona remove it if there are not tests for this part :)